polyfill/lib/calendar.mjs

```diff
@@ -2034,10 +1986,9 @@ impl['buddhist'] = ObjectAssign({}, nonIsoGeneralImpl, { helper: helperBuddhist
 impl['japanese'] = ObjectAssign({}, nonIsoGeneralImpl, { helper: helperJapanese });
 impl['gregory'] = ObjectAssign({}, nonIsoGeneralImpl, { helper: helperGregory });
 
-function calendarImpl(calendar) {
+export function calendarImpl(calendar) {
```

Comment:
Only `ecmascript.mjs` has to access it that way, to avoid a circular import. The other files can just import it normally. (I've been wanting to untangle that for a long time, because it messes up other things. But today is not that day.)

Reply:
It's no longer exported now that we use `ES.calendarImplForObj()` in the other files.